src/battle/other_battle_calculators.c

```diff
@@ -3079,7 +3079,43 @@ BOOL LONG_CALL ov12_02251A28(struct BattleSystem *bsys, struct BattleStruct *ctx
         // There’s no PP left for this move!
         msg->msg_id = BATTLE_MSG_CANNOT_USE_MOVE_NO_PP;
         ret = FALSE;
+    } 
+
+#ifdef DEBUG_ENABLE_UNIMPLEMENTED_MOVES
+    else if (ctx->moveTbl[ctx->battlemon[battlerId].move[movePos]].flag & FLAG_UNUSABLE_UNIMPLEMENTED) {
+        msg->msg_tag = TAG_NICKNAME_MOVE;
+        // Reusing Disable for now
+        // {STRVAR_1 1, 0, 0}’s {STRVAR_1 6, 1, 0}\nis disabled!\r
+        msg->msg_id = BATTLE_MSG_CANNOT_USE_MOVE_DISABLED;
+        msg->msg_para[0] = CreateNicknameTag(ctx, battlerId);
```

Comment:
```suggestion
        // Reusing Disable for now
        // {STRVAR_1 1, 0, 0}’s {STRVAR_1 6, 1, 0}\nis disabled!\r
        msg->msg_id = 0; // empty message
        msg->msg_para[0] = CreateNicknameTag(ctx, battlerId);
```

Same for below cases.

https://youtu.be/O27O5TL4ZKU?t=5m17s